e2e/e2e-local.sh

```diff
@@ -7,15 +7,21 @@ set -euxo pipefail
 : "${LOCATION:=eastus}"
 : "${CLUSTER_NAME:=agentbaker-e2e-test-cluster}"
 
+# Error out if SCENARIO_NAME, VM_SKU, or SCENARIO_NAME are unbound
```

Comment:
SIG_VERSION instead of SCENARIO_NAME?

Error out if SCENARIO_NAME, VM_SKU, or SIG_VERSION are unbound

Reply:
woops good catch